openstack/compute/v2/extensions/quotasets/fixtures.go

```diff
@@ -0,0 +1,59 @@
+// +build fixtures
+
+package quotasets
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	th "github.com/rackspace/gophercloud/testhelper"
+	"github.com/rackspace/gophercloud/testhelper/client"
+)
+
+// GetOutput is a sample response to a Get call.
+const GetOutput = `
+{
+   "quota_set" : {
+      "instances" : 25,
+      "security_groups" : 10,
+      "security_group_rules" : 20,
+      "cores" : 200,
+      "injected_file_content_bytes" : 10240,
+      "injected_files" : 5,
+      "metadata_items" : 128,
+      "ram" : 200000,
+      "keypairs" : 10,
+      "injected_file_path_bytes" : 255
+   }
+}
+`
+
+const FirstTenantID = "555544443333222211110000ffffeeee"
+
+// FirstQuotaset is the first result in ListOutput.
+var FirstQuota = Quotaset{
```

Comment:
The type being called `QuotaSet` is fine. I was merely noting that in `requests_test.go`, you're referencing a variable named `FirstQuotaSet`, which doesn't exist and is causing the unit test to fail. Renaming this variable to `FirstQuotaSet` would likely fix that.